modules/querying/pages/func/edge-methods.adoc

```diff
@@ -28,8 +28,14 @@ The data type of the attribute itself.
 | Parameter | Description | Data type
 
 | `attrName`
-| The name of the attribute.
-| `STRING`
+| The name of an attribute. This can be a query parameter, constant string, or global string variable.
+| Base types: `STRING` `INT` `UINT` `DATETIME` `DOUBLE` `FLOAT` `BOOL`
+
+Container types: `LIST<INT/DATETIME/DOUBLE/STRING>`, `SET<INT/DATETIME/DOUBLE/STRING>`
```

Comment:
Could you use `LIST<INT>`, `LIST<DATETIME>`, `LIST<DOUBLE>`, `LIST<FLOAT>`
and `SET<INT>`, `SET<DATETIME>`, `SET<DOUBLE>`, `SET<FLOAT>`